### ISSUE TITLE
Failed to Resolve with index.tsx files

### ISSUE THREAD

**Issue by robertoyoc** [antoine-coulon/skott#120]
With the following structure:

```
  src/common/components/SafeContainer
├── constants.ts
├── elements.tsx
├── index.test.tsx
└── index.tsx
```
Then in another file I have:

```
import { SafeContainer } from "@/common/components/SafeContainer";
```
This is my tsconfig for the aliasing:
```
{
  "compilerOptions": {
    "baseUrl": ".",
    "jsx": "react",
    "esModuleInterop": true,
    "paths": {
      "@/*": ["./src/*"],
      "@assets/*": ["./assets/*"]
    },
    "strict": true
  },
  "exclude": ["node_modules", "dist"]
}
```
After running `skott index.js --verbose` I see

```
Start resolution for src/common/components/SafeContainer 
 ✖ Failed to resolve src/common/components/SafeContainer/index.js 

 ✖ Failed to resolve src/common/components/SafeContainer.ts 

 ✖ Failed to resolve src/common/components/SafeContainer/index.ts 

 ✖ Failed to resolve s.ts 

 ✖ Failed to resolve src/common/components/SafeContainer.js 

 ✖ Failed to resolve src/common/components/SafeContainer.tsx 

 ✖ Failed to resolve src/common/components/SafeContainer.jsx 

 ✖ Failed to resolve src/common/components/SafeContainer/index.js 
 
```
Looks like library is not checking index.tsx/index.jsx paths?
Thanks for your help.

**Comment by antoine-coulon** [comment]
@robertoyoc thanks for opening that issue! It's indeed correct, skott is not checking `index.jsx/index.tsx` modules yet. TSX/JSX support was added incrementally, and this is something I completely forgot.

**Comment by antoine-coulon** [comment]
Fixed in #121 by @robertoyoc, will be released in `0.31.3` 🎉

@all-contributors please add @robertoyoc for code
